packages/eas-cli/src/graphql/queries/EnvironmentVariablesQuery.ts

```diff
@@ -0,0 +1,130 @@
+import { print } from 'graphql';
+import gql from 'graphql-tag';
+
+import { ExpoGraphqlClient } from '../../commandUtils/context/contextUtils/createGraphqlClient';
+import { withErrorHandlingAsync } from '../client';
+import {
+  EnvironmentVariableEnvironment,
+  EnvironmentVariableFragment,
+  EnvironmentVariablesByAppIdQuery,
+} from '../generated';
+import { EnvironmentVariableFragmentNode } from '../types/EnvironmentVariable';
+
+export const EnvironmentVariablesQuery = {
+  async byAppIdWithSensitiveAsync(
+    graphqlClient: ExpoGraphqlClient,
+    {
+      appId,
+      environment,
+      filterNames,
+    }: {
+      appId: string;
+      environment: EnvironmentVariableEnvironment;
+      filterNames?: string[];
+    }
+  ): Promise<{
+    appVariables: EnvironmentVariableFragment[];
+  }> {
+    const data = await withErrorHandlingAsync(
+      graphqlClient
+        .query(
+          gql`
+            query EnvironmentVariablesIncludingSensitiveByAppId(
+              $appId: String!
+              $filterNames: [String!]
+              $environment: EnvironmentVariableEnvironment!
+            ) {
+              app {
+                byId(appId: $appId) {
+                  id
+                  environmentVariablesIncludingSensitive(
+                    filterNames: $filterNames
+                    environment: $environment
+                  ) {
```

Comment:
If we get rid of sudo entirely then yes, but if we decide to implement it anyway it will be useful to have `environmentVariablesIncludingSensitive` and `environmentVariables` separate.